Scheduler - Getting Started: Add ASP.NET Core Snippets (#9038)
DevExpress/devextreme-documentation: concepts/05 UI Components/Scheduler/00 Getting Started with Scheduler/25 Time Zone Support.md

#include common-tutorialbutton-named with { url: "/Documentation/Guide/UI_Components/Scheduler/Time_Zone_Support/", name: "Time Zone Support" }

> The Scheduler displays all appointments in the client time zone. To change this default behavior, specify the [timeZone](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/timeZone.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#timeZone') property. This option accepts [IANA time zone identifiers](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones). This tutorial sets the time zone to "Europe/Berlin".

Reviewer: ```suggestion
Scheduler displays all appointments in the client time zone by default. To use a different time zone, specify the [timeZone](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/timeZone.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#timeZone') property. This option accepts [IANA time zone identifiers](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones). This tutorial sets the time zone to "Europe/Berlin".
```
